presto-main/src/main/java/com/facebook/presto/type/VarbinaryOperators.java

```diff
@@ -92,7 +92,7 @@ public static long hashCode(@SqlType(StandardTypes.VARBINARY) Slice value)
         // This needs to match the hash function for VARBINARY blocks
         // (i.e. AstractVariableWidthBlock.hash(...))
         // TODO: we need to get rid of hash from Block and rely on HASH_CODE operators only
-        return XxHash64.hash(value);
+        return value.longHashCode();
```

Comment:
Just a question. Does the comment above "This needs to match the hash function for VARBINARY blocks" still hold?